src/AccessibilityInsights.Actions/Actions/ListenAction.cs

```diff
@@ -42,73 +36,43 @@ public class ListenAction:IDisposable
         /// <param name="config"></param>
         /// <param name="ec"></param>
         /// <param name="listener"></param>
-        private ListenAction(RecorderSetting config, ElementContext ec, HandleUIAutomationEventMessage listener)
+        private ListenAction(TreeScope listenScope, ElementContext ec, HandleUIAutomationEventMessage listener)
```

Comment:
Ended up renaming the enum itself from TreeScope to ListenScope to match the config field name - so now the enum + variable name are consistent. Do LMK if your comment indicated a preference for treeScope itself rather than the inconsistency of the enum + variable name